docs/src/user/script.rst

```diff
@@ -54,6 +54,11 @@ the local one passed to
 created. To access the particular working directory of a trial, see next sections
 :ref:`commandline_templates` and :ref:`env_vars`.
 
+Language compatibility
+======================
+The command line works for scripts and programs in any language.
+The only requirement is that the executed script returns a JSON string with the objective value.
+
```

Comment:
The format is
```
[
    {
        'name': 'some-objective',
        'type': 'objective',
        'value': 1
    }
]
```

See https://orion.readthedocs.io/en/latest/code/client.html#orion.client.report_results for more details.

Reply:
Very good!